src/NGettext/Catalog.cs

```diff
@@ -182,12 +182,14 @@ private string _FindTranslationFile(CultureInfo cultureInfo, string domain, stri
 
 		private string _GetFileName(string localeDir, string domain, string locale)
 		{
-			var relativePath =

```

Comment:
Why do you change this?
It is not mono weirdness, it's just the project is compatible with .net 2.0 whitch does not have this Path.Combine overload.